arith/src/extension_field.rs

```diff
@@ -1,10 +1,14 @@
+use std::ops::Mul;
+
 use crate::{Field, FieldSerde};
 
 /// Configurations for Extension Field over
 /// - either the Binomial polynomial x^DEGREE - W
 /// - or the AES polynomial x^128 + x^7 + x^2 + x + 1
 //
-pub trait ExtensionField: From<Self::BaseField> + Field + FieldSerde {
+pub trait ExtensionField:
+    Mul<Self::BaseField> + From<Self::BaseField> + Field + FieldSerde
```

Comment:
I think it's better to have `Add` as well if we have `Mul` here, but I'm fine with this.